src/sale.rs

```diff
@@ -539,7 +517,7 @@ impl Contract {
         let mut sale: Sale = self.sales.get(&sale_id).expect("ERR_NO_SALE").into();
 
         let account_id = env::predecessor_account_id();
-        let distribute_token_decimals = sale.distribute_token_decimals.expect("ERR_NO_TOKEN_DECIMALS");
+        let _distribute_token_decimals = sale.distribute_token_decimals.expect("ERR_NO_TOKEN_DECIMALS");
```

Comment:
Please consider renaming to distribute_token_decimals_unwrapped or distribute_token_decimals_value, underscore prefix is typically using for ignored variables

Reply:
wil be renamed to distribute_token_decimals_value